LayoutTests/fast/harness/user-preferred-language.html

```diff
@@ -19,8 +19,8 @@
 
                 test('internals.userPreferredLanguages returns a non-empty array', languages.length);
 
-                languages.unshift("first-language");
```

Comment:
I don't quite understand this. How come this change doesn't seem to be accompanied by a change in an -expected.txt file?

Reply:
I tried to explain this in the change log but basically, the test sets the override languages to these values, then queries them again and checks that setting worked. There is no change in expected.txt because the test doesn't actually print the locales, it only checks that the written locales and the read ones are the same.

And the reason we need to use proper languages instead of "first-language" is that we now store these in AppleLanguages UserDefault and not in a plain Vector in WTF, and then read them using CF APIs. If we don't use real languages, they get dropped and we don't see them via the CF APIs.

Reply:
🆒